tests/integration/update.bats

```diff
@@ -123,6 +124,34 @@ function check_cgroup_value() {
         check_cgroup_value $CGROUP_MEMORY "memory.memsw.limit_in_bytes" 96468992
     fi
 
+    if [ -f "$CGROUP_MEMORY/memory.memsw.limit_in_bytes" ]; then
```

Comment:
this test can be merge into the conditional above right, make it clearer to see that we have an existing memory-swap limit too.